dc-cudami-server/dc-cudami-server-webapp/src/main/java/de/digitalcollections/cudami/server/controller/identifiable/entity/ProjectController.java

```diff
@@ -129,11 +129,10 @@ public PageResponse<Project> findAll(
 
   @Operation(summary = "Get all projects as list")
   @GetMapping(
-      value = {"/v5/projectlist", "/v2/projectlist", "/latest/projectlist"},
+      value = {"/v6/projects/all", "/v5/projectlist", "/v2/projectlist", "/latest/projectlist"},
```

Comment:
Warum dieser `/all` Suffix? Gemäß REST wäre der Endpunkt ja wie in v5 einfach nur `/projectlist`?

Reply:
stimmt, wird entfernt